client/src/components/common/ExportFiles.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Menu, MenuButton, MenuList, MenuItem, Button } from '@chakra-ui/react';
+
+const ExportFiles = (props) => {
```

Comment:
Add the destructuring back in the parameters. It's almost always better to specify explicitly what the function is expecting.

Reply:
Fixed